dds_web/api/schemas/project_schemas.py

```diff
@@ -153,24 +147,9 @@ def create_project(self, data, **kwargs):
                     }
                 )
             )
-            # Generate keys
-            # NOTE: TEMPORARY:
-            # key=b"temp" * 8 is only a temporary KEK until we have the actual KEK
-            aesgcm = cryptography.hazmat.primitives.ciphers.aead.AESGCM(key=b"temp" * 8)
-            nonce = os.urandom(12)
-            project_private_key = bytes.fromhex(new_project.private_key)
-            aad = None
-            # ----
-
-            # Add new project keys to table.
-            new_project_key = models.ProjectKeys(
-                key=aesgcm.encrypt(nonce=nonce, data=project_private_key, associated_data=aad),
-                nonce=nonce,
-            )
 
-            # Save
-            auth.current_user().project_keys.append(new_project_key)
-            new_project.project_keys.append(new_project_key)
+            generate_project_key_pair(current_user, new_project)
```

Comment:
We need to add pproject key rows for all unitusers

Reply:
done in the last commit